src/App.jsx

```diff
@@ -14,57 +15,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  return (
   <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+    {selectedGood ? (
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood} is selected
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
+          />
+        )}
+      </h1>
+    ) : (
+      <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+    )}
+
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
 
     <table className="table">
       <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+        {goods.map(good =>(
```

Comment:
Using the index as a key in a list can lead to issues, especially if the list can change. Consider using a unique identifier for each item in the 'goods' array instead.